test/callAPIMethod.spec.js

```diff
@@ -164,5 +164,36 @@ test.serial('unsupported Response method', async t => {
     t.true(result instanceof TypeError);
     t.false(result instanceof APIError);
 
+    fetchMock.restore();
+});
+
+test.serial('returns full response object', async t => {
+    fetchMock.get(matcher, Response200);
+
+    const APINamespace = 'rest-api';
+    const namespace = 'user';
+    const fetchOptions = { method: 'POST'};
+    const methodOptions = { path: 'path' };
+    const responseOptions = { fullResponse: true };
+
+    const result = await callAPIMethod(APINamespace, namespace, fetchOptions, methodOptions, responseOptions);
+    const mockedResult = {
+        body: '{"sample":"data"}',
```

Comment:
By default we have [type](https://github.com/auru/unity-api#type-string-optional) of response equals `json`.
We there we have a string?
[Example](url)